website/docs/r/log_analytics_data_export_rule.html.markdown

```diff
@@ -60,6 +60,14 @@ The following arguments are supported:
 
 * `enabled` - (Optional) Is this Log Analytics Data Export Rule enabled? Possible values include `true` or `false`. Defaults to `false`.
 
+* `metadata` - (Optional) A `metadata` block as defined below.
+
+---
+
+The `metadata` block supports the following attributes:
+
+* `eventhub_name` - (Optional) The name of the EventHub that is defined for the Log Analytics Data Export Rule.
```

Comment:
as above, what is this actually doing? if this is setting anything? just a place to put the destination event hub name? should it just be
```suggestion
* `destination_eventhub_name` - (Optional) The name of the EventHub that Log Analytics Data Export Rule exports too.
```

also given we are giving the id above? what is the point of this? could we not just grab it from the ID and set it ourselves so the user doesn't have to?

Reply:
Closed this comment as duplicate